### PR TITLE
feat: add new connection statuses

### PR DESCRIPTION
**Pull request by CedrikNikita**
fixes https://github.com/superhero-com/superhero-wallet/issues/3026

### CODE REVIEW

**Comment by Liubov-crypto** [comment]
LGTM

File: src/popup/components/ConnectionStatus.vue [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<template>
+  <transition name="fade-transition">
+    <div
+      v-if="status"
+      data-cy="connect-node"
+      class="node-connection-status"
```
Comment (martinkaintas) [inline src/popup/components/ConnectionStatus.vue:6]:
```suggestion
      class="connection-status"
```

File: src/popup/components/ConnectionStatus.vue [hunk 2]
```diff
@@ -0,0 +1,79 @@
+<template>
+  <transition name="fade-transition">
+    <div
+      v-if="status"
+      data-cy="connect-node"
```
Comment (martinkaintas) [inline src/popup/components/ConnectionStatus.vue:5]:
```suggestion
      data-cy="connect-status"
```

**Review by martinkaintas — APPROVED** [review]
In general LGTM, left some small suggestions

**Pull request merged**